apps/web/src/app/(main)/dashboard/newsletters/[slug]/page.tsx

```
@@ -0,0 +1,149 @@
+                </span>
+              </span>
+              <span className="flex items-center gap-1">
+                <ClockIcon className="size-4 shrink-0" />
+                <span className="whitespace-nowrap">{newsletter.readTime}</span>
+              </span>
+            </div>
+          </header>
+
+          <div
+            className="newsletter-content"
+            dangerouslySetInnerHTML={{ __html: newsletter.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)